package.nls.json

```diff
@@ -143,6 +144,8 @@
     "AWS.message.prompt.quickStart.toastMessage": "You are now using the AWS Toolkit for Visual Studio Code, version {0}",
     "AWS.message.prompt.region.hide.title": "Select a region to hide from the AWS Explorer",
     "AWS.message.prompt.region.show.title": "Select a region to show in the AWS Explorer",
+    "AWS.message.prompt.copyButtonLabel": "Copy",
+    "AWS.message.toolkitInfo": "AWS.message.toolkitInfo','OS:  {0} {1} {2}\nVisual Studtio Code Version:  {3}\nAWS Toolkit for Visual Studio Code Version:  {4}\n",
```

Comment:
Assuming `AWS.message.toolkitInfo','` is not meant to be in this file?

Reply:
Typo: 'Studtio'